front/js/cart.js

```diff
@@ -1,92 +1,324 @@
+    productQtty.setAttribute("name", "itemQuantity")
 
     // Insertion de l'élément "div"
-    let productItemContentSettingsDelete = document.createElement("div");
-    productItemContentSettings.appendChild(productItemContentSettingsDelete);
-    productItemContentSettingsDelete.className = "cart__item__content__settings__delete";
+    let productItemContentSettingsDelete = document.createElement("div")
+    productItemContentSettings.appendChild(productItemContentSettingsDelete)
+    productItemContentSettingsDelete.className = "cart__item__content__settings__delete"
 
     // Insertion de "p" supprimer
-    let productSupprimer = document.createElement("p");
-    productItemContentSettingsDelete.appendChild(productSupprimer);
-    productSupprimer.className = "deleteItem";
-    productSupprimer.innerHTML = "Supprimer";
+    let productSupprimer = document.createElement("p")
+    productItemContentSettingsDelete.appendChild(productSupprimer)
+    productSupprimer.className = "deleteItem"
+    productSupprimer.innerHTML = "Supprimer"
 }
 }}
-getCart();
\ No newline at end of file
+getCart()
+
+function getTotals(){
+
+    // Récupération du total des quantités
+    let elementQuantity = document.getElementsByClassName('itemQuantity')
+    let elementLength = elementQuantity.length,
+    totalQtt = 0
+
+    for (let index = 0; index < elementLength; ++index) {
+        totalQtt += elementQuantity[index].valueAsNumber
+    }
+
+    let productTotalQuantity = document.getElementById('totalQuantity')
+    productTotalQuantity.innerHTML = totalQtt
+    console.log(totalQtt)
+
+    // Récupération du prix total
+    totalPrice = 0;
+
+    for (let index = 0; index < elementLength; ++index) {
+        totalPrice += (elementQuantity[index].valueAsNumber * productLocalStorage[index].prodcutPrice)
+    }
+
+    let productTotalPrice = document.getElementById('totalPrice')
+    productTotalPrice.innerHTML = totalPrice
+    console.log(totalPrice)
+}
+getTotals()
+
+function modifyQtt() {
+    let elementModif = document.querySelectorAll(".itemQuantity")
+
+    for (let modif = 0; modif < elementModif.length; modif++){
+        elementModif[modif].addEventListener("change" , (event) => {
+            event.preventDefault()
+
+            //Selection de l'element à modifier en fonction de son id ET sa couleur
+            let quantityModif = productLocalStorage[modif].productQuantity
+            let elementModifValue = elementModif[modif].valueAsNumber
+            
+            const resultFound = productLocalStorage.find((element) => element.elementModifValue !== quantityModif)
+
+            resultFound.productQuantity = elementModifValue
+            productLocalStorage[modif].productQuantity = resultFound.productQuantity
+
+            localStorage.setItem("product", JSON.stringify(productLocalStorage))
+        
+            // refresh rapide
+            location.reload()
+        })
+    }
+}
+modifyQtt()
```

Comment:
this shouldn't be called like that out of the blue... it should be a function called by a event (change) on the input where you changed the number of items. for instance what you do is really brutal because it go over each items and get it's quantity whereas you could register a change event on each html inputs and when value change you it call your function with as a entry parameter the item value as property which mean you don't need to go over each elements